pkg/crc/cluster/pullsecret.go

```diff
@@ -78,7 +77,7 @@ func NewNonInteractivePullSecretLoader(config crcConfig.Storage, path string) Pu
 
 func (loader *nonInteractivePullSecretLoader) Value() (string, error) {
 	// If crc is built from an OKD bundle or podman bundle is used, then use the fake pull secret in contants.
```

Comment:
Can you fix the 'contants' typo while you are changing code in that area?

Reply:
Ack, will do.